### DISCUSSION TITLE
Multiple enex file processing with version 4.2

### DISCUSSION THREAD

**Discussion by pmottishaw** [akosbalasko/yarle#316] [General]
I just used the new multiple enex file capability of v4.2 and it saves a lot of time. I tested it on a notebook that produced 5 enex files of around 80 Mbytes each and contained around 200 notes with multiple attachments. Once the obsidian files are generated I can merge the 5 folders in a couple of steps using search in File Manager on Windows 10.

Most of the internal links (i.e. between the 200 notes) work fine (I chose the include .md extension option which solves some problems with attachments being confused with notes). Problems occur when 

1. I had changed the name of the note in Evernote. This is a lot of work to fix and there is an open [issue](https://github.com/akosbalasko/yarle/issues/292#issuecomment-908407888)  to address it.
2. The note title contained characters (e.g. < > \  / : ) that are illegal in file names. It would be nice to fix this by replacing with some legal character.

From earlier experiments I think multiple notebooks would work equally well because I seem to have no Evernote notes with duplicate titles. Obsidian does not mind about the path to a note as long as the name is unique.

**Comment by akosbalasko** [comment]
Hi @pmottishaw !

Thanks for the feedback, I put effort to fix the issues you raised.

Cheer,
akos
